docs/user_guide/index.rst

```diff
@@ -9,6 +9,7 @@ User Guide
    fairness_in_machine_learning
    assessment
    mitigation
+   thresholder
```

Comment:
I have moved it to mitigation.rst, in the Postprocessing part. I did not see `ThresholdOptimizer` there (because it is still in progress I guess?), so could not place it in front of that. Also, I added a small description in the table at the top of the page. I added a check mark for regression of which I’m not entirely sure: `Thresholder` can take a regressor as input, but the output will always be binary, so not sure if this should be a check mark or a cross?

Reply:
I agree, regression probably doesn't make sense. 

The ThresholdOptimizer PR is still in progress and I won't get to it for at least a few weeks. I'm basically blocked on a few other issues (like the synthetic data PR) which I'd like to rely on for that PR. I suppose I could do the same as you and use this dataset instead. But by all means go ahead and put your user guide into the postprocessing section! That's perfectly fine. I'll adjust around it. I think yours is more foundational anyway.